_rules/image-button-non-empty-accessible-name-59796f.md

```diff
@@ -56,6 +56,8 @@ Each target element has an [accessible name][] that is neither empty (`""`), nor
 
 There is a known combination of a popular browser and assistive technology that does not by default support `title` as an [accessible name][].
 
+The image button with non-empty `title` attribute but empty `alt` attribute results in a button with empty accessible name on many browsers. On Firefox, instead, the `title` attribute is used as fallback, resulting in a button with `title` attribute value as accessible name. 
```

Comment:
I think we should try to write this without naming any specific browsers (like we do in the previous sentence).

Reply:
Thanks Carlos,
updated using a generic _known popular browser_  as above.